From bf636176bb6d4f4042cdb045c33d620e39978e0b Mon Sep 17 00:00:00 2001
From: Yong Zhao <yong.zhao@amd.com>
Date: Wed, 14 Sep 2016 16:13:05 -0400
Subject: [PATCH 1645/1722] Revert "drm/amdgpu: Disable certain CG on Carrizo
 to workaround KFD suspend/resume"

This reverts commit c97ee7bb2164 ("drm/amdgpu: Disable certain CG
on Carrizo to workaround KFD suspend/resume").

The implication is that suspend/resume is broken on Carrizo in
favour of SHOCBenchmark to pass.

Change-Id: Ifa4569a3086f24afc83eb03dee26592ba0f97942
Signed-off-by: Yong Zhao <yong.zhao@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/vi.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/vi.c b/drivers/gpu/drm/amd/amdgpu/vi.c
index ee32f38..c15734f 100644
--- a/drivers/gpu/drm/amd/amdgpu/vi.c
+++ b/drivers/gpu/drm/amd/amdgpu/vi.c
@@ -974,8 +974,9 @@ static int vi_common_early_init(void *handle)
 			AMD_CG_SUPPORT_GFX_RLC_LS |
 			AMD_CG_SUPPORT_GFX_CP_LS |
 			AMD_CG_SUPPORT_GFX_CGTS |
+			AMD_CG_SUPPORT_GFX_MGLS |
 			AMD_CG_SUPPORT_GFX_CGTS_LS |
-			/*AMD_CG_SUPPORT_GFX_CGCG |*/
+			AMD_CG_SUPPORT_GFX_CGCG |
 			AMD_CG_SUPPORT_GFX_CGLS |
 			AMD_CG_SUPPORT_BIF_LS |
 			AMD_CG_SUPPORT_HDP_MGCG |
-- 
2.7.4

